src/clusto/services/http.py

```diff
@@ -93,9 +93,22 @@ def setattr(self, request):
         Set an attribute on this object.
 
         Requires HTTP parameters "key" and "value"
-        Optional parameters are "subkey" and "number"
+        Optional parameters are "subkey", "number" and "datatype"
```

Comment:
do we need to revert? "datatype" is a new optional parameter, should be documented.